packages/lesswrong/components/review/ReviewVoteTableRow.tsx

```diff
@@ -216,6 +216,9 @@ const ReviewVoteTableRow = ({ post, dispatch, costTotal, classes, expandedPostId
   // note: this needs to be ||, not ??, because quadraticScore defaults to 0 rather than null
   const userReviewVote = post.currentUserReviewVote?.quadraticScore || qualitativeScoreDisplay;
 
+  const visitedDate = markedVisitedAt || post.lastVisitedAt
```

Comment:
I realize this is just moving existing code, but the `||` should probably be a `??` (if it's a `Date` rather than a `number` where we want the 0 value to get coalesced).